schedule/virt_autotest/kubevirt-tests-agent.yaml

```diff
@@ -0,0 +1,29 @@
+name:           kubevirt-tests-agent
```

Comment:
Could you help also move this file also into a 'kubevirt folder' in data/virt_autotest if this belongs to kubevirt or something else ? You should know that the data/virt_autotest folder starts growing, it would be better to curb potential chaos before it becomes true.

Reply:
What I understand is that the scheduler file for test module arrangement should be placed under schedule/xxx folder. It's not reasonable to place it into data folder, right?

Reply:
Yes, you are right. It should be in schedule folder. So your current placement is temporary ?